pagerduty-trigger

```diff
@@ -15,18 +15,20 @@ if [ -s $CONFIG_FILE ]; then
   . /etc/pagerduty/pagerduty.conf
 fi
 
-if [ -z "$1" ]; then
-  echo "Usage: $0 <event>"
-  exit 1
-elif [ -z "$PAGERDUTY_SERVICE_KEY" ]; then
+if [ -z "$PAGERDUTY_SERVICE_KEY" ]; then
   echo "Failed to trigger event: you must set the PAGERDUTY_SERVICE_KEY variable."
   exit 1
 elif [ ! -x "/usr/local/bin/pagerduty" ]; then
   echo "Failed to trigger event: /usr/local/bin/pagerduty does not exist or is not executable."
```

Comment:
Sounds like a good thing to implement. Feel free to add a card to our backlog and me and @simpsora can prioritise.